lte/cloud/go/services/policydb/obsidian/handlers/policy_handlers.go

```diff
@@ -337,7 +337,7 @@ func UpdateRule(c echo.Context) error {
 	if err := rule.ValidateModel(reqCtx); err != nil {
 		return echo.NewHTTPError(http.StatusBadRequest, err)
 	}
-	if ruleID != string(rule.ID) {
+	if ruleID != string(*rule.ID) {
```

Comment:
why is rule id  now a pointer?

Note policy rule is used across the system so a change like that may impact other CPP services like sessiond

Reply:
We don't completely understand why some fields (like rule ID and subprofile) became pointers in the generated code after the swaggergen library upgrade. However, we didn't change anything in the model so I don't think it should have any impact on CPP services.